hera_pspec/pspecdata.py

```diff
@@ -864,44 +905,40 @@ def R(self, key):
 
         # Only add to Rkey if a particular mode is enabled
         # If you do add to this, you need to specify this in self.set_R docstring!
-        Rkey = key + (self.data_weighting,) + (self.taper,)
+        Rkey = key + (self.taper,)
         if self.data_weighting == 'dayenu':
             # add extra dayenu params
             Rkey = Rkey + tuple(self.filter_extension,) + (self.spw_Nfreqs,) \
                    + (self.symmetric_taper,)
 
+        Rkey = Rkey + (self.data_weighting,)
```

Comment:
I see you added it here downstream--probably still good to keep it as is, because having `dayenu` in the key will be easier for poeple to understand what it is anyways. and ordering of `data_weighting`, `taper` is what is in the `set_R` doc string

Reply:
it sounds like this is alright so I will keep the code as is.